Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/UAudioManagerBase.cs

```diff
@@ -12,9 +12,12 @@ namespace HoloToolkit.Unity
     /// </summary>
     /// <typeparam name="TEvent">The type of AudioEvent being managed.</typeparam>
     /// <remarks>The TEvent type specified must derive from AudioEvent.</remarks>
-    public partial class UAudioManagerBase<TEvent> : MonoBehaviour where TEvent : AudioEvent, new()
+    public partial class UAudioManagerBase<TEvent, TBank> : MonoBehaviour where TEvent : AudioEvent, new() where TBank : AudioBank<TEvent>, new()
     {
+        public TBank[] DefaultBanks = null;
+
         [SerializeField]
+        [System.Obsolete]
```

Comment:
What replaces this?
What is the new usage?
Why didn't the usages get updated int he `EditorEvents` down below?